cmd/deployment-check/deployment-check.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: kh-deployment-check
+  namespace: kuberhealthy
+spec:
+  runInterval: 30m
+  timeout: 5m
+  podSpec:
+    containers:
+    - name: kh-deployment-check
+      image: <image>
+      imagePullPolicy: IfNotPresent
+      env:
+        - name: CHECK_DEPLOYMENT_REPLICAS
+          value: "6"
+        - name: CHECK_TIME_LIMIT_SECONDS
+          value: "300"
+        - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
+          value: "true"
+        - name: SHUTDOWN_GRACE_PERIOD_SECONDS
+          value: "30"
+      resources:
+        requests:
```

Comment:
consider adding limits here for cpu (maybe not memory to prevent unnecessary crashes)

Reply:
Added a limit here, but only for cpu